.github/workflows/block-extractor-scheduler.yml

```diff
@@ -0,0 +1,76 @@
+name: Block Extractor Workflow
+
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+
+env:
+  RUST_LOG: info
+  GCP_BLOCK_EXTRACTOR_SA_KEY: ${{ secrets.GCP_BLOCK_EXTRACTOR_SA_KEY }}
+
+jobs:
+  backup-blocks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        database: [postgres, bigquery]
+        network: [mainnet, testnet]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Compile evm-block-extractor
+        run: cargo build --release -p evm-block-extractor --bin evm-block-extractor-server
+
+      - name: Move binary to root
+        run: mv target/release/evm-block-extractor-server evm-block-extractor-server
+
+      - name: Make binary executable
+        run: chmod +x evm-block-extractor-server
+
+      - name: Set RPC URL and Database Config
+        run: |
+          if [ "${{ matrix.network }}" = "mainnet" ]; then
+              echo "rpc_url=mainnet.bitfinity.network" >> $GITHUB_ENV
+          elif [ "${{ matrix.network }}" = "testnet" ]; then
+              echo "rpc_url=testnet.bitfinity.network" >> $GITHUB_ENV
+          fi
+
+          if [ "${{ matrix.database }}" = "bigquery" ]; then
+              echo "db_project_id=${{ secrets.BIGQUERY_PROJECT_ID }}" >> $GITHUB_ENV
+              echo "db_dataset_id=${{ secrets.BIGQUERY_DATASET_ID }}" >> $GITHUB_ENV
+
+          elif [ "${{ matrix.database }}" = "postgres" ]; then
+              echo "db_username=${{ secrets.POSTGRES_USERNAME }}" >> $GITHUB_ENV
+              echo "db_password=${{ secrets.POSTGRES_PASSWORD }}" >> $GITHUB_ENV
```

Comment:
printing the password?!!?!? :0 
please remove it.
I would remove this entire set of prints anyway. The executables print the required information at startup

Reply:
Not really! This command does not directly print the value of the secrets, GitHub secrets are encrypted environment variables created in a repository and can only be used in GitHub Actions workflows. The value is passed safely

Reply:
Done